api/proxy/servers/arbitrum_altda/cli.go

```diff
@@ -0,0 +1,51 @@
+package arbitrum_altda
+
+import (
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	ListenAddrFlagName = "arbitrum-da.addr"
+	PortFlagName       = "arbitrum-da.port"
+	Enabled            = "arbitrum-da.enabled"
+)
+
+func withEnvPrefix(prefix, s string) []string {
+	return []string{prefix + "_ARB_CUSTOM_DA_" + s}
+}
+
+func CLIFlags(envPrefix string, category string) []cli.Flag {
+	flags := []cli.Flag{
+		&cli.StringFlag{
+			Name:     ListenAddrFlagName,
+			Usage:    "Server listening address",
+			Value:    "0.0.0.0",
+			EnvVars:  withEnvPrefix(envPrefix, "ADDR"),
+			Category: category,
+		},
+		&cli.IntFlag{
+			Name:     PortFlagName,
+			Usage:    "Server listening port",
+			Value:    3100,
+			EnvVars:  withEnvPrefix(envPrefix, "PORT"),
+			Category: category,
+		},
+		&cli.BoolFlag{
+			Name:     Enabled,
+			Usage:    "Whether or not to enable Arbitrum Custom DA JSON RPC API",
```

Comment:
curious why you use `custom` everywhere? is that part of arbitrum's spec? it sounds bespoke and not very serious. can we just name is arbitrum altda json rpc api?